### PR TITLE
remove userEmail from license request tool

### PR DESCRIPTION
**Pull request by jlovejoy**
accompanying changes to #379 

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Comment by jlovejoy** [comment]
does "userEmail" also need to be removed on line 150?

**Review by goneall — APPROVED** [review]
@jlovejoy - Thanks for pulling this together - you're now an official DJango programmer ;)

Were you able to test these out on your local machine?  There are 2 ways to setup a local environment to run this on your local computer.  That way, you can check out it out visually before it is fully deployed.

**Review by goneall — CHANGES_REQUESTED** [review]
@jlovejoy Please ignore my last comment - that was intended for your other PR (clicked on the wrong tab).

Could you merge in PR #379 into this PR?  That way we can test if it works before merging.

**Comment by jlovejoy** [comment]
@goneall how do I merge #379 into this PR? Can I do that from the web interface? this seems like an advanced move...

**Comment by goneall** [comment]
> how do I merge https://github.com/spdx/spdx-online-tools/pull/379 into this PR? Can I do that from the web interface? this seems like an advanced move...

Hmm - don't know about the web interface.  I only use that for the minutes.  From the command line, you would setup a remote for the branch the other PR was using then merge from that branch - a bit of git magic involved.  You could also just copy/paste the changes if they are small enough.  BTW - You're already in the advanced class doing some Django / Python coding ;)

**Comment by jlovejoy** [comment]
@goneall - I can just add the other changes here, that's easier. Speaking of which/in that case - would it be better to simply add all of the remove-email address field changes to #387 ?

**Comment by rtgdk** [comment]
@jlovejoy Yes, you can add this patch branch on top of the patch branch of #387, just need to change the base branch of this PR from `master` to desired one

I have merged #379 into this PR by changing the base branch to `jlovejoy-patch-2` so you can test end-to-end and also changed the base branch for this PR to `jlovejoy-patch-1`. Now we just need to merge this PR first and then #387 .

File: src/app/views.py [hunk 1]
```diff
@@ -117,7 +117,6 @@ def submitNewLicense(request):
                     licenseHeader = form.cleaned_data['licenseHeader']
                     licenseComments = form.cleaned_data['comments']
                     licenseText = form.cleaned_data['text']
-                    userEmail = form.cleaned_data['userEmail']
```
Comment (rtgdk) [inline src/app/views.py:120]: You need to remove all other instances of this variable `userEmail` used in the below code.
Like line no 149
```
data['userEmail'] = userEmail
```
then line 163
```
licenseRequest = LicenseRequest(licenseAuthorName=licenseAuthorName, fullname=licenseName, shortIdentifier=licenseIdentifier,
                        submissionDatetime=now, userEmail=userEmail, notes=licenseNotes, xml=xml)
```

Comment (rtgdk) [reply src/app/views.py:120]: @jlovejoy You can make `userEmail=""` (empty) or update the `LicenseRequest` model on 163 to pass userEmail as None/Empty. We can also change the underlying structure of `LicenseRequest` but let's not do it right now.

**Review by rtgdk — CHANGES_REQUESTED** [review]

**Comment by rtgdk** [comment]
@jlovejoy Requested some changes. then we can merge this on #387  and test everything together.

File: src/app/views.py (hunk 1 above)
Comment (jlovejoy) [reply src/app/views.py:120]: see latest changes...

**Comment by goneall** [comment]
@jlovejoy @rtgdk - Just checking on the status of this PR - going to do a release over the next few days

**Comment by jlovejoy** [comment]
I think this is ready (at least from my somewhat naive perspective) @goneall 

thanks for the help @rtgdk 

I do still need to do more work (or some help) on the other one tho, but might be good to merge this first?

**Comment by rtgdk** [comment]
@jlovejoy @goneall LGTM.
Merging this in `master` instead of `jlovejoy-patch-1` #387

**Comment by jlovejoy** [comment]
Thanks @rtgdk ! #387 is a separate set of changes, so I'll try to get back to working on that now!

**Pull request merged**